python/paddle/fluid/framework.py

```diff
@@ -1199,6 +1199,14 @@ def _to_readable_code(self):
         if self.persistable:
             var_str = "persist " + var_str
 
+        from paddle.distributed.auto_parallel.context import get_default_distributed_context
```

Comment:
why not import at the beginning of file

Reply:
Should be corrected. But this may be removed in the future.